css-four-column-layout/styles.css

```diff
@@ -0,0 +1,98 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 1200px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-fourth {
+  width: 25%;
+}
+
+.background-color-lgrey {
+  background-color: #f7f8fa;
+}
+
+.justify-center {
+  display: flex;
```

Comment:
You can leave off the `display: flex` here, and just add the `row` class _and this class_ to the element.